nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/HealthCheckService.java

```diff
@@ -116,10 +118,12 @@
     private final AtomicInteger threadNumber = new AtomicInteger(1);
 
     private ScheduledExecutorService executor;
-    private final Map<String, HealthCheckTask> registeredTasks = new HashMap<String, HealthCheckTask>();
+    private ScheduledExecutorService historicCleanerExecutor;
```

Comment:
argh another threadpool! 
Longer term we need to consolidate all these Executors. into a single general service.